hello.txt

```diff
@@ -0,0 +1,60 @@
+hello this is a message from brackets...
+
+Ok. I checked out feature3 branch before writing this. Are we really only on 3?
+
+Sprucing it up with a few Hamilton lyrics:
+
+ELIZA:
+Un deux trois quatre

```

Comment:
Very funny! I think you can handle a few numbers. If not, just enjoy the rest. : )